### PR TITLE
Points: Add InterleavedBufferAttribute support to raycast.

### PR DESCRIPTION
**Pull request by manthrax**
Make Points raycaster aware of InterleavedBufferAttribute by fixing hard coded vertex indexing.

### CODE REVIEW

File: src/objects/Points.js [hunk 1]
```diff
@@ -79,17 +81,17 @@ Points.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 					const a = indices[ i ];
 
-					_position.fromArray( positions, a * 3 );
+					_position.fromArray( positions, a * stride + offset );
```
Comment (Mugen87) [inline src/objects/Points.js:84]: It's atually better to use `Vector3.fromBufferAttribute()` in order to hide the interleaved logic.

Comment (manthrax) [reply src/objects/Points.js:84]: I ran some informal performance tests based on some feedback from @usefulthink ..
On a 5 million random point cloud:
fromArray with inline address calculation took min 65 to max 81   msec
fromBufferAttribute with getX/Y/Z            took min 79 to max 110 msec
fromBufferAttribute optimized with a .toVector3 method on BufferAttribute and InterleavedBufferAttribute for faster conversion to Vector3                                took min 70 to max 93 msec.

Any opinions/guidance on what approach to take @Mugen87 ?

Comment (manthrax) [reply src/objects/Points.js:84]: Option 3 is in between both methods in terms of speed, but requires adding a .toVector3 method to BufferGeometry and InterleavedBufferGeometry...

Comment (Mugen87) [reply src/objects/Points.js:84]: Sorry, but performance is not the most important point here. It's code readability and consistency.  Hence, please use `Vector3.fromBufferAttribute()`.

Comment (mrdoob) [reply src/objects/Points.js:84]: Indeed. Unless it was 10x slower or something... These numbers seem like it's readability should win here.

Comment (manthrax) [reply src/objects/Points.js]: Cool. Thanks ya'll! :)

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**